src/lib.rs

```diff
@@ -687,3 +687,19 @@ impl BlitPipeline {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    fn assert_is_send<T: Send>() {}
+
+    #[test]
+    fn check_scene_is_send() {
+        assert_is_send::<super::Scene>();
+    }
+
+    #[test]
+    #[cfg(all(feature = "wgpu", not(target_arch = "wasm32")))]
+    fn check_renderer_is_send() {
+        assert_is_send::<super::Renderer>();
```

Comment:
I'm not certain whether we definitely want this, but we should at least make changing this be intentional, so this check is good.

Reply:
It is required for how the `with_bevy` sample is written.